starboard/build/config/BUILD.gn

```diff
@@ -210,9 +210,7 @@ config("starboard") {
       defines += [ "SB_IS_EVERGREEN_COMPATIBLE_LIBUNWIND=1" ]
     }
 
-    defines += [
-      "STARBOARD_CONFIGURATION_INCLUDE=\"$starboard_path/configuration_public.h\"",
-    ]
+    defines += [ "STARBOARD_CONFIGURATION_INCLUDE=\"$starboard_path/configuration_public.h\"" ]
```

Comment:
I don't know if i agree with this but if that's what `gn format` spits out okay

Reply:
definitely harder to read but it is what gn format decrees.

Reply:
this should be the last time we have to do this as this PR got in before lintting / pre-committing got in.